narwhal/source_code_sharing/update_liferay_dxp_source_code_of_hotfixes.sh

```diff
@@ -178,22 +172,46 @@ function checkout_commit {
 	fi
 }
 
+function checkout_hotfix_tag {
+	local base_branch_name="${3}"
+	local repository="${1}"
+	local temporary_branch_name="${2}"
+
+	trap 'return ${LIFERAY_COMMON_EXIT_CODE_BAD}' ERR
+
+	lc_cd "${BASE_DIR}/${repository}"
+
+	git reset --hard -q
+
+	git clean -dfqX
+
+	git checkout -q -b "${temporary_branch_name}" "${base_branch_name}"
```

Comment:
@lucasmiranda0, can we sort the parameters here? I mean `... -b "${temporary_branch_name}" -q`.